src/Personal.cpp

```diff
@@ -6,7 +6,7 @@
 #include "StdAfx.h"
 #include "glpers.h"
 
-#define AT_Log(...) AT_Log_I("Personal", __VA_ARGS__)
+#define AT_Log(...) // AT_Log_I("Personal", __VA_ARGS__)
```

Comment:
A particular reason on why you removed the logs here?

Reply:
Most of the logging in this file was added by me two years ago when I implemented the "re-create employees in the hiring pool when not hired for a long time". I think this feature is now working well and the logging was a bit too verbose, to be honest. So I disabled it again.